docs/content/release_notes.rst

```diff
@@ -1,6 +1,20 @@
 Release Notes
 =============
 
+Release 0.6.1 - 2021/12/09
+
+**(Minor Release)**
+
+This release mostly addresses issues encountered since release of v0.6.0.
+
+ - Remove json warnings from jupyter-notebook apps.
+ - Optimization and bug fixes for Peak Finder
+ - Fix crash at the end of multi-component inversions
+ - Fix update upper/lower bound dropdowns on geoh5 change.
+ - Remove 'Z' options from dropdown channels. Leave empty if vertices are to be used.
+ - General API maintenance and unit tests
+
```

Comment:
- remove redundant checkboxes for data channel selection in inversion ui.json files.

Reply:
Done